<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdate">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>390</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Cập nhật</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>450</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Quay lại</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tên Phim</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtReleaseDate">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ngày Sản Xuất</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRating">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Điểm Đánh Giá</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="giai">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>240</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Giải Thưởng</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dienvien">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Diễn Viên</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dao">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Đạo Diễn </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tenphim">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tên Phim</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="mota">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>330</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Mô tả </string>
     </property>
    </widget>
    <widget class="QLabel" name="anhthu">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>50</y>
       <width>150</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="chonhinh">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>chọn ảnh </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="binhluan">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>420</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Bình Luận</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="luotxem">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>510</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Lượt Xem</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="thich">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>540</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Lượt Thích</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboGenre_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ratingComboBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1★</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2★</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3★</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4★</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5★</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ẤN 2 LẦN VÀO PHIM ĐÃ CHỌN ĐỂ XOÁ NỘI DUNG TRONG Ô</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1274</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
